community_tasks/french_evals.py

```diff
@@ -117,5 +129,23 @@ def prompt_gpqa_fr(line, task_name: str = None):
     version=0,
 )
 
+# BAC-fr task
+bac_fr_task = LightevalTaskConfig(
+    name="bac-fr",
+    suite=["community"],
+    prompt_function=prompt_bac_fr,
+    hf_repo="fr-gouv-coordination-ia/bac-fr",
+    hf_subset="default",
+    hf_avail_splits=["train"],
+    evaluation_splits=["train"],
+    few_shots_split=None,
+    few_shots_select="random_sampling",
+    generation_size=1,
+    metric=[],  # To be defined
```

Comment:
If we feel like the instructions constrain the answer enough, we could go for an exact match. We can also look at @hynky1999 's parser for math equations.

Reply:
This is match bench ?

Reply:
Includes math questions

Reply:
(but not math only, I'd say 1/3 to 1/2 are math questions?)